src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -98,7 +100,11 @@ export const InitializeClient: FC<OwnProps> = ({
   }, [bucket, onSelectBucket])
 
   useEffect(() => {
-    dispatch(getAllResources())
+    try {
+      dispatch(getAllResources())
```

Comment:
Same here - could you post a screenshot to show this notifying the user as expected on success / error?